src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1160,23 +1161,64 @@ private void NextScheduledActivityCallback(string databaseName, IdleDatabaseActi
                             break;
 
                         case IdleDatabaseActivityType.WakeUpDatabase:
-                            _ = TryGetOrCreateResourceStore(databaseName, nextIdleDatabaseActivity.DateTime).ContinueWith(t =>
+                            if (_serverStore.ConcurrentBackupsCounter.CanRunBackup == false)
```

Comment:
```
var delayInMs = _dueTimeOnRetry + Random.Shared.Next(0, _dueTimeOnRetry);
```

use this instead of a const, by doing that you will delay from 60 to 120s

Reply:
done